kaldb/src/main/java/com/slack/kaldb/logstore/search/KaldbDistributedQueryService.java

```diff
@@ -119,11 +119,13 @@ private void updateStubs() {
   }
 
   private List<SearchResult<LogMessage>> distributedSearch(KaldbSearch.SearchRequest request) {
+    LOG.info("Starting distributed search for request: {}", request);
     ScopedSpan span =
         Tracing.currentTracer().startScopedSpan("KaldbDistributedQueryService.distributedSearch");
 
     try {
       List<ListenableFuture<SearchResult<LogMessage>>> queryServers = new ArrayList<>(stubs.size());
+      span.tag("stub_count", String.valueOf(stubs.size()));
```

Comment:
Tagging this as a stub count is somewhat awkward, since stubs is just a grpc specific detail. What if we referred to this as `queryServerCount`? We should also change this to camel cased, since all the other tags use that format.